src/components/Wishlist/Wishlist.jsx

```diff
@@ -0,0 +1,12 @@
+import {Main, Header, Footer} from "./";
+const Wishlist = () => {
+    return (
+        <div className="grid-3-rows w-100">
+            <Header/>
```

Comment:
If you are adding Header and Footer in every component, instead of that you could add them to the App.jsx file and render them regardless of the route. Could reduce reptition

Reply:
You don't need to add header and footer to every component just add it to App.js